### PR TITLE
Work and about features included

### PR DESCRIPTION
**Pull request by 0zedd**
Added work and about sections to the mobile website portfolio

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,14 +30,74 @@ <h1 class="info_box">
...
+            <div>
+                <ul class="sec_h">
+                    <li><a href="#"><img src="canopy.svg" alt="canopy"></a></li>
+                    <li><a href="#"><img src="Count.svg" width="1" height="5" alt="count"></a></li>
+                    <li><a href="#"><img src="bed.svg" height="22" alt="bed"></a></li>
+                    <li><a href="#"><img src="cc.svg" width="1" height="5" alt="dot"></a></li>
+                    <li><a href="#"><img src="time.svg" alt="date"></a></li>
+                </ul>
+            </div>
+       </header>
```
Comment (Mre55) [inline index.html]: Kindly, remove the `header` tag here we don't need it rather we can use either the `article` tag or just a `div`. And I see in all of the images `src` you used no existing images and in the page, it's not rendering anything please add images first then use a proper path for you all images.

https://user-images.githubusercontent.com/34578631/170739762-6fc30354-e4ac-46a5-9865-979e98dd5f7f.mov

File: index.html [hunk 2]
```diff
@@ -30,14 +30,74 @@ <h1 class="info_box">
...
+            </h3>
+            <div>
+                <ul class="push">
+                    <li><a href="#">html</a></li>
+                    <li><a href="#">CSS</a></li>
+                    <li><a href="#">Javascript</a></li>
+                </ul>
+            </div>
+            <h3 class="endpush"><a href="#">See Project</a></h3> 
+        </section>
```
Comment (Mre55) [inline index.html]: Please stick to the Figma design provided as much as possible and also add three more works here and make them look like the given design here I put your design along with the Figma one.

| Your Design | Figma Design |
|--------------|---------------|
| image | image |

File: index.html [hunk 3]
```diff
@@ -30,14 +30,74 @@ <h1 class="info_box">
...
+            <div>
+                <ul class="dropim">
+                    <li class="dropim"><a href="#">languages</a></li>
+                    <li class="dropim"><a href="#">Javascript</a></li>
+                    <li class="dropim"><a href="#">HTML</a></li>
+                    <li class="dropim"><a href="#">CSS</a></li>
+                    <li class="fr_sk">Frameworks</li>
+                    <li class="fr_sk">Skills</li>
+                </ul>
+            </div>
```
Comment (Mre55) [inline index.html]: The same comment goes here as the project section stick to the Figma design.

| Your Design | Figma Design |
|--------------|--------------|
| image | image |

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @0zedd,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️🛠️ 
_Check the comments under the review._





## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -10,14 +10,14 @@
...
+        <a href="#"><img src="/img/menu.svg" alt="Menu"></a>
     </div>
-    <img class="arc" src="arc.svg" width="1" height="15" alt="arc">
-    <img class="bar" src="bar.svg" width="1" height="15" alt="bar">
-    <img class="c" src="c.svg" width="1" height="15" alt="curve">
-    <img class="circle" src="circle.svg" width="1" height="15" alt="circle">
+    <img class="arc" src="/img/arc.svg" width="1" height="15" alt="arc">
+    <img class="bar" src="/img/bar.svg" width="1" height="15" alt="bar">
+    <img class="c" src="/img/c.svg" width="1" height="15" alt="curve">
+    <img class="circle" src="/img/circle.svg" width="1" height="15" alt="circle">
```
Comment (jaflih) [inline index.html]: Good job for changing the path. But it still does not work. For correct it do like this (`add . before /img`):
(do this for all your image)
```
<img class="arc" src="./img/arc.svg" width="1" height="15" alt="arc">
    <img class="bar" src="./img/bar.svg" width="1" height="15" alt="bar">
    <img class="c" src="./img/c.svg" width="1" height="15" alt="curve">
    <img class="circle" src="./img/circle.svg" width="1" height="15" alt="circle">

```

File: index.html [hunk 5]
```diff
@@ -30,14 +30,186 @@ <h1 class="info_box">
...
+        <section>
+            <h3 class="p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </h3>
+            <div>
+                <ul class="push">
+                    <li class="html"><a href="#">html</a></li>
+                    <li class="css"><a href="#">CSS</a></li>
+                    <li class="javasc"><a href="#">Javascript</a></li>
+                </ul>
```
Comment (jaflih) [inline index.html]: As said the previous reviewer, please stick to Figma:
On figma to find what styles to use, just click on the element, and on the right you will see the CSS. property (size, color ...) 

your
image

Figma:
image

File: style.css [hunk 6]
```diff
@@ -109,3 +104,268 @@ ul {
...
+  font-size: 16px;
+  margin-top: -5px;
+  height: 140px;
+  gap: 24px;
+  padding: 114px, 24px, 114px, 24px;
+  line-height: 24px;
+  font-family: 'Poppins', sans-serif;
+  color: #344563;
+  width: 300px;
+}
```
Comment (jaflih) [inline style.css]: Please stick to Figma, like said the previous reviewer

image

Figma:
image

File: index.html [hunk 7]
```diff
@@ -30,14 +30,186 @@ <h1 class="info_box">
...
+    </section>
+
+
+    <section class="about_section_container">
+        <h2 class="about_my">About Myself</h2>
+        <h4 class="sec_abt">
+            Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </h4>
+        <div>
+            <p class="contact_section"><a href="#">LET'S CONNECT</a></p>
```
Comment (jaflih) [inline index.html]: Please stick to FIgma , here use the good color

image

Figma
image

File: style.css [hunk 8]
```diff
@@ -109,3 +104,268 @@ ul {
...
+  grid-template-rows: repeat(6, 2fr);
+  row-gap: 40px;
+  font-size: 20px;
+  color: #253858;
+  margin-left: 36px;
+  margin-right: 36px;
+  font-weight: 500;
+  font-family: 'Poppins', sans-serif;
+  line-height: 24px;
+}
```
Comment (jaflih) [inline style.css]: Like ask the previous reviewer , please stick to Figma

image

Figma
image

File: index.html [hunk 9]
```diff
@@ -30,14 +30,186 @@ <h1 class="info_box">
...
+            <div class="tonic">
+                <h2>Tonic</h2>
+            </div>
+            <div>
+                <ul class="sec_h">
+                    <li><a href="#"><img src="/img/canopy.svg" alt="canopy"></a></li>
+                    <li><a href="#"><img src="/img/Count.svg" width="1" height="5" alt="count"></a></li>
+                    <li><a href="#"><img src="/img/bed.svg" height="22" alt="bed"></a></li>
+                    <li><a href="#"><img src="/img/cc.svg" width="1" height="5" alt="dot"></a></li>
+                    <li><a href="#"><img src="/img/time.svg" alt="date"></a></li>
```
Comment (jaflih) [inline index.html]: [OPTIONAL] instead of using image, write the text and design it using CSS to make your project more professional

**Review by jaflih — CHANGES_REQUESTED** [review]
Hi @0zedd 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.




 ## Highlights

The Pull Request is following the GitHub Flow ✅
The README is well documented ✅
Linters are working properly ✅

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — APPROVED** [review]
Hi @0zedd, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**